### ISSUE TITLE
.net8 isolated functions - timeout on startup

### ISSUE THREAD

**Issue by MrAntix** [corvus-dotnet/Corvus.Testing#447]
Hello, 

I am running a spec-flow feature using Corvus-Testing on .net8 using the isolated functions worker.
The worker starts and runs, but hits this after 60s

https://github.com/corvus-dotnet/Corvus.Testing/blob/main/Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/FunctionsController.cs#L130

This seems to be because corvus doesn't think the host is running, ie it never gets any messages matching these
https://github.com/corvus-dotnet/Corvus.Testing/blob/main/Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/Internal/FunctionOutputBufferHandler.cs#L55

Does this require specific logging to be on?

Has there been a change to the worker host in .net8 from "Hosting Started" to "Host Started (xxxms)" - I can see the later in the output of a console window, but not the former.

Can you suggest a work around?

Many thanks

**Comment by jesshill99** [comment]
We just tried to reproduce this by setting up a .net8 isolated function. We are still seeing the "Hosting Started" in the output of the console window.

We included this `local.settings.json` so that we can verify the message appears when the function is configured in the same way that Corvus.Testing configures it:

```json
{
  "IsEncrypted": false,
  "Values": {
    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
    "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
    "AzureFunctionsJobHost:logging:logLevel:default": "Debug"
  }
}
```

The final setting: `"AzureFunctionsJobHost:logging:logLevel:default": "Debug"` is important as without this, we do not get the message.

We do still see this message on .net8.0, so then we tried launching it with a `FunctionsController` 

```cs
var fcnet8 = new FunctionsController(NullLogger.Instance);

await fcnet8.StartFunctionsInstance(@"C:\Users\YourUserNameHere\source\repos\Net6InProcess\Net8Isolated\", 7135, "net8.0");

Console.WriteLine("Started");

Console.ReadLine();

fcnet8.TeardownFunctions();
```
And we verified that this was still responding to a http request.

Is it possible that you're doing something with configuration that somehow locks the `"AzureFunctionsJobHost:logging:logLevel:default": "Debug"` setting the `FunctionsController` tries to supply? It does that by setting it as an environment variable, so if you have changed how .NET is reading its configuration settings maybe that is why this isn't working?
